libs/components/core/src/lib/modules/trim/trim.directive.ts

```diff
@@ -9,18 +9,19 @@ import { MutationObserverService } from '../mutation/mutation-observer-service';
   selector: '[skyTrim]',
 })
 export class SkyTrimDirective implements OnInit, OnDestroy {
-  #obs: MutationObserver;
+  #elRef: ElementRef;
 
-  constructor(
-    private elRef: ElementRef,
-    private mutationObs: MutationObserverService
-  ) {
-    this.#obs = this.mutationObs.create((mutations: MutationRecord[]) => {
+  #obs: MutationObserver | undefined;
```

Comment:
Why is this declared as `| undefined` when it's set in the constructor?

Reply:
We had a line in the `#disconnect` method that set the `#obs` to `undefined`. I removed the `| undefined` and the `undefined` assignment and all is well.